### PR TITLE
Implement Windows (WinFSP) version of mountpoint

### PR DESCRIPTION
**Pull request by touilleMan**

### CODE REVIEW

**Review by Max-7 — APPROVED** [review]

File: Cargo.toml [hunk 1]
```diff
@@ -187,7 +187,8 @@ web-sys = { version = "0.3.69", default-features = false }
 widestring = { version = "1.0.2", default-features = false }
 windows-sys = { version = "0.52.0", default-features = false }
 winfsp_build = { version = "0.1.1", default-features = false }
-winfsp_wrs = { version = "0.1.0", git = "https://github.com/Scille/winfsp_wrs.git", rev = "317b700", default-features = false }
+# winfsp_wrs = { version = "0.1.0", git = "https://github.com/Scille/winfsp_wrs.git", rev = "317b700", default-features = false }
+winfsp_wrs = { version = "0.1.0", git = "https://github.com/Scille/winfsp_wrs.git", rev = "1affd1090702c95964d9f276077b2ec7dc933beb", default-features = false }
```
Comment (TimeEngineer) [inline Cargo.toml:191]: Remove comment

File: bindings/generator/api/common.py [hunk 2]
```diff
@@ -39,7 +39,7 @@ class EnumItemUnit:
 #         class A:
 #             a: int
 #         B = VariantItemTuple(int, int)
-#         C = VariantItemUnit
+#         C = VariantItemUnit()
```
Comment (TimeEngineer) [inline bindings/generator/api/common.py:42]: Why this change ?

File: libparsec/crates/platform_mountpoint/src/lib.rs [hunk 3]
```diff
@@ -1,27 +1,55 @@
...
+#[cfg(target_os = "windows")]
 mod windows;
 
 #[cfg(target_os = "linux")]
 pub(crate) use unix as platform;
-#[cfg(any(target_os = "windows", target_os = "macos"))]
+#[cfg(target_os = "windows")]
 pub(crate) use windows as platform;
 
+// TODO: UNIX implementation is linux-only for the moment
```
Comment (TimeEngineer) [inline libparsec/crates/platform_mountpoint/src/lib.rs:13]: Update comment

File: libparsec/crates/platform_mountpoint/tests/unit/operations/windows_winfsp_tests.rs [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+use std::{path::PathBuf, process::Command, sync::Arc};
+
+use libparsec_client::{Client, WorkspaceOps};
+use libparsec_tests_fixtures::prelude::*;
+
+use super::utils::mount_and_test;
+
+#[parsec_test(testbed = "minimal_client_ready")]
+// // TODO: Currently failing :'(
+// #[ignore]
```
Comment (TimeEngineer) [inline libparsec/crates/platform_mountpoint/tests/unit/operations/windows_winfsp_tests.rs:14]: Remove these comments

File: libparsec/crates/platform_mountpoint/src/lib.rs (hunk 3 above)
Comment (touilleMan) [reply libparsec/crates/platform_mountpoint/src/lib.rs:13]: This comment is still relevant: macOS is not supported for the FUSE implementation (cf. the `use std::{os::linux::fs::MetadataExt, sync::Arc, thread::JoinHandle};` in `mount.rs`)

File: libparsec/crates/platform_mountpoint/tests/unit/operations/windows_winfsp_tests.rs (hunk 4 above)
Comment (touilleMan) [reply libparsec/crates/platform_mountpoint/tests/unit/operations/windows_winfsp_tests.rs]: done.

File: Cargo.toml (hunk 1 above)
Comment (touilleMan) [reply Cargo.toml]: done.

**Review by TimeEngineer — APPROVED** [review]
Looks good after tests pass
